src/Temporalio/Worker/WorkflowReplayer.cs

```diff
@@ -20,13 +20,21 @@ namespace Temporalio.Worker
     /// </summary>
     public class WorkflowReplayer
     {
+        private readonly IReadOnlyCollection<ITemporalWorkerPlugin> plugins;
+
         /// <summary>
         /// Initializes a new instance of the <see cref="WorkflowReplayer"/> class. Note, some
         /// options validation is deferred until replay is attempted.
         /// </summary>
         /// <param name="options">Replayer options.</param>
         public WorkflowReplayer(WorkflowReplayerOptions options)
         {
+            plugins = options.Plugins ?? new List<ITemporalWorkerPlugin>();
```

Comment:
```suggestion
            plugins = options.Plugins ?? Array.Empty<ITemporalWorkerPlugin>();
```

Pedantic, but a bit better option here IIUC